### PR TITLE
feat(@ciscospark/media-engine-webrtc): add serialize function

### PR DESCRIPTION
**Pull request by adamweeks**

### CODE REVIEW

File: packages/node_modules/@ciscospark/media-engine-webrtc/src/engine.js [hunk 1]
```diff
@@ -686,6 +686,15 @@ export default class WebRTCMediaEngine {
   triggerNegotiationNeeded() {
     this.trigger(`negotiationneeded`);
   }
+
+  /**
+   * Returns a string when attempting to serialize object
+   * (Applies mostly this is a property of AmpState object)
```
Comment (ianwremmel) [inline packages/node_modules/@ciscospark/media-engine-webrtc/src/engine.js]: wording?

File: packages/node_modules/@ciscospark/media-engine-webrtc/src/engine.js [hunk 2]
```diff
@@ -686,6 +686,15 @@ export default class WebRTCMediaEngine {
...
+   * @returns {String}
```
Comment (ianwremmel) [inline packages/node_modules/@ciscospark/media-engine-webrtc/src/engine.js]: lowercase

File: packages/node_modules/@ciscospark/media-engine-webrtc/src/engine.js (hunk 1 above)
Comment (adamweeks) [reply packages/node_modules/@ciscospark/media-engine-webrtc/src/engine.js]: Deep. I re-edited that three times. Guess it wasn’t a charm

Comment (adamweeks) [reply packages/node_modules/@ciscospark/media-engine-webrtc/src/engine.js]: Derp

**Pull request merged**